src/main/resources/wa-task-cancellation-sscs-benefit.dmn

```diff
@@ -256,6 +256,29 @@
           <text>"reviewUrgentHearingRequest"</text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_0gfe5m1">
+        <inputEntry id="UnaryTests_1pbpfql">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0bp5hhr">
+          <text>"voidCase", "appealWithdrawn", "appealDormant", "confirmLapsed", "struckOut", "issueFinalDecision", "decisionIssued"</text>
```

Comment:
"struckOut" should be a completion event.

Reply:
Thanks Mark, now fixed.